src/Components/AddNewBook.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable linebreak-style */
+function AddNewBook() {
+  return (
+    <form>
+      <input placeholder="Book Title" />
+      <input placeholder="Book Author" />
```

Comment:
- [ ] KIndly include the type="text" to your input fields 👍🏽

Reply:
Done. Thank you